src/components/WorkPoolCombobox.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <p-combobox v-model="internalValue" :options="options" :empty-message="emptyMessage">
+    <template #combobox-options-empty>
+      No work pools
+    </template>
+    <template #default="scope">
+      <slot v-bind="scope" />
+    </template>
+    <template #option="{ option }">
+      <slot name="option" :option="option" />
+    </template>
+  </p-combobox>
+</template>
+
+<script lang="ts" setup>
+  import { PCombobox, SelectOption } from '@prefecthq/prefect-design'
+  import { useSubscription } from '@prefecthq/vue-compositions'
+  import { computed } from 'vue'
+  import { useWorkspaceApi } from '@/compositions'
+
+  const props = defineProps<{
+    selected: string | string[] | null | undefined,
+    emptyMessage?: string,
+    allowUnset?: boolean,
+  }>()
+
+  const emits = defineEmits<{
+    (event: 'update:selected', value: string | string[] | null): void,
+  }>()
+
+  const multiple = computed(() => Array.isArray(props.selected))
```

Comment:
The behavior for the `p-select` (which I think is what `p-combobox` uses under the hood) multiple attribute changed a bit recently - is this still necessary? I was under the impression it wasn't but I haven't worked with this much

Reply:
I took it from WorkQueueCombobox since we actively used it before this update, so I don't have a good answer to this question. Who could be a good person to shed some light on how it works now?

Reply:
@stackoverfloweth should know; I _think_ you should be able to ignore the attribute entirely here and let callers pass it themselves as an attr (which the combobox will inherit)

Reply:
^ that was my hope as well, if so we could simplify the model values on lots of these comboboxes

Reply:
Nice! I'll make an issue for that, to go through combo boxes and clean them up

Reply:
I don't know that *for sure* we have extra complexity, just something that's worth exploring